ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/security/KsqlDefaultSecurityExtension.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Confluent Inc.
```

Comment:
nit: year. Same comment for `KsqlSecurityExtension.java` below.

Reply:
Done